### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by axefrog** [anaisbetts/starter-mobile#14]
Hi, as best I can tell, this starter kit is only designed to work on Mac. Any chance of getting it to compile out of the box on Windows? I've tried on both VS2013 and Xamarin for Windows and both have issues.

**Comment by anaisbetts** [comment]
Yep, this is generally only designed to work on Mac. I'd take a PR for Windows support if it was relatively unobtrusive though
